java/apps/recovery-portal/src/main/webapp/errors/error_401.jsp

```diff
@@ -30,13 +32,18 @@
 <%-- Branding Preferences --%>
 <jsp:directive.include file="../includes/branding-preferences.jsp"/>
 
+<%
+    String stat = IdentityManagementEndpointUtil.i18n(recoveryResourceBundle, "error.401");
+%>
+
 <%-- Data for the layout from the page --%>
 <%
-    layoutData.put("containerSize", "medium");
+    layoutData.put("isResponsePage", true);
+    layoutData.put("isErrorResponse", true);
 %>
 
 <!doctype html>
-<html lang="en-US">
+<html>
```

Comment:
Let's add the lang attribute back, as it's required for i18n.

Reply:
https://github.com/wso2/identity-apps/pull/6249/commits/dd9575ae867c6b5e9f648b34523c0db6f01875b6